docs/ops.html

```diff
@@ -1129,6 +1128,26 @@ <h4><a id="remote_jmx" href="#remote_jmx">Security Considerations for Remote Mon
         <td>kafka.server:type=BrokerTopicMetrics,name=ReassignmentBytesInPerSec</td>
         <td></td>
       </tr>
+      <tr>
+        <td>Size of a partition on disk (in bytes)</td>
+        <td>kafka.log:type=Log,name=Size,topic=([-.\w]+),partition=([0-9]+)</td>
+        <td>The size of a partition on disk, measured in bytes.</td>
+      </tr>
+      <tr>
+        <td>Number of log segments in a partition</td>
+        <td>kafka.log:type=Log,name=NumLogSegments,topic=([-.\w]+),partition=([0-9]+)</td>
+        <td>The number of log segments in a partition.</td>
+      </tr>
+      <tr>
+        <td>First offset in a partition</td>
+        <td>kafka.log:type=Log,name=LogStartOffset,topic=([-.\w]+),partition=([0-9]+)</td>
+        <td>The first offset in a partition.</td>
+      </tr>
+      <tr>
+        <td>Last offset in a partition</td>
+        <td>kafka.log:type=Log,name=LogEndOffset,topic=([-.\w]+),partition=([0-9]+)</td>
+        <td>The last offset in a partition.</td>
```

Comment:
Is this the last committed offset? Or, is it uncommitted offsets? Meaning, if a producer appends a record there but the followers haven't yet acked it, will this value get incremented?

Does that level of detail matter, in the docs?

Reply:
Tag @hachikuji

Reply:
This is the last uncomitted offset: after the local append it would be incremented, not waiting for other replicas to ack.